RetailCoder.VBE/Inspections/InspectionBase.cs

```diff
@@ -81,6 +83,28 @@ protected virtual IEnumerable<Declaration> BuiltInDeclarations
             get { return State.AllDeclarations.Where(declaration => declaration.IsBuiltIn); }
         }
 
+        protected bool HasIgnoreAnnotation(VBComponent component, int line)
```

Comment:
No, I do not think they could be combined--they are not similar at all.  However, I could look at moving the declaration one into InspectionBase.